app/scripts/lib/errors.js

```diff
@@ -136,6 +136,16 @@ define(function (require, exports, module) {
       return error.errno === code;
     },
 
+    /**
+     * Check if an error was created by this module.
+     *
+     * @param {object} error - error to check
+     * @returns {boolean} - true if from this module, false otw.
+     */
+    createdError: function (error) {
```

Comment:
Bikeshedding massively here, sorry, but what do you think of just calling this method `is`? When you look at how it's invoked, that would read quite nicely I reckon:

```js
  if (MarketingEmailErrors.is(err)) {
    // ...
  }
```